### ISSUE TITLE
Found bug when parsing @returns

### ISSUE THREAD

**Issue by Codevendor** [Codevendor/inferjs-compiler#6]
**Describe the bug**
Syntax error, when missing text with @return param.

### Causes Error
```js
@returns 
or
@returns {object}
```
### No Error
```js
@returns {object} - Returns object
```
Need to investigate.

**Comment by Codevendor** [comment]
This will be fixed in new parser i'm building

**Comment by Codevendor** [comment]
Fixed in v0.0.9
